src/Google.Api.Gax/project.json

```diff
@@ -7,6 +7,9 @@
     "projectUrl": "htttps://github.com/googleapis/gax-dotnet",
     "licenseUrl": "https://developers.google.com/open-source/licenses/bsd",
     "tags": [ "Google" ],
+    "compilationOptions": {
+        "keyFile": "../../Gax.snk"

```

Comment:
It will when this PR is committed - it's the first file shown above, but with "binary file not shown" :)